app/Keycloak/Client/KeycloakClient.php

```diff
@@ -4,25 +4,10 @@
 
 namespace App\Keycloak\Client;
 
-use App\Keycloak\Config;
-use GuzzleHttp\ClientInterface;
-use GuzzleHttp\Exception\GuzzleException;
-use GuzzleHttp\Psr7\Uri;
 use Psr\Http\Message\RequestInterface;
 use Psr\Http\Message\ResponseInterface;
 
-final readonly class KeycloakClient
+interface KeycloakClient
```

Comment:
I know, but separation of concern wise I thought having 2 classes would be nice.
Do you have a strong preference for changing it?

Reply:
Both classes send requests to Keycloack, so I think they have the same concern. I think it would make the bootstrapping, injection, and testing easier with one interface with two methods.